runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
Removing the code in `XWalkBrowserContext::GetMediaRequestContextForStoragePartition` for the non-Android case can potentially break commit 507013f772a8ab688aec2c68e671ab0c8e449038, can't it? To me it's similar to https://codereview.chromium.org/1892893002 for `chrome/browser/profiles/profile_impl.cc`. The Android code path may be fine with returning `nullptr`, though even in that case I'm not entirely sure.